src/main/java/rs/onako2/iwie/entity/CreakingEntity.java

```diff
@@ -28,6 +41,37 @@ public static DefaultAttributeContainer.Builder createMobAttributes() {
                 .add(EntityAttributes.GENERIC_ATTACK_DAMAGE, 3.0);
     }
 
+    @Override
+    public void tick() {
+        super.tick();
+        if(this.getWorld().isClient()) {
+            return;
+        }
+        if(this.isHeartSpawn && this.getWorld().isDay() || (this.isHeartSpawn && !Util.isHeartNear(this, this.getWorld(), 16))) {
```

Comment:
Hi!
Seems like checking every tick if there is a creaking heart makes the server lagg.
![grafik](https://github.com/user-attachments/assets/b3262811-8327-4a98-a221-3a917c6e89bf)

Theoretically we could store in the nbt the coordinates and then check if that heart is still there and if not then check?

Reply:
Ok, let me try this solution.

Reply:
Now I changed the code according to the solution you provided, and I updated it to my fork repository, but now I don’t know how to pull the request again on this page